projects/ngx-maplibre-gl/src/lib/map/map.service.spec.ts

```diff
@@ -208,13 +209,100 @@ describe('MapService', () => {
       popupOpen: { emit: jasmine.createSpy() },
       popupClose: { emit: jasmine.createSpy() },
     } as any;
-    const popup = mapService.createPopup({ 
-      popupOptions: {}, 
+    const popup = mapService.createPopup({
+      popupOptions: {},
       popupEvents
     }, container);
     mapService.addPopupToMap(popup, [0, 0]);
     mapService.removePopupFromMap(popup);
     popup.fire('close');
     expect(popupEvents.popupClose.emit).not.toHaveBeenCalled();
   });
+
+  describe('layer handling', () => {
```

Comment:
Is there a way to make this year simpler but still relevant to the issue? The test code is a bit too complicated for me...

Reply:
Is there a way to remove some of event handlers initializations?